.eslintrc.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   root: true,
-  extends: "babel",
+  parser: "lightscript-eslint",
```

Comment:
tldr version: do a clean install outside the path of any other code you're working on, try to run the unit tests, then you will understand this change :)